apps/ncs_digital_sign/Makefile

```diff
@@ -0,0 +1,106 @@
+GREEN = '\033[1;32m'
+YELLOW = '\033[1;33m'
+NOCOLOR = '\033[0m'
+
+.PHONY: all
+all: deps data
+
+FACE_RELATIVE_DIR = ../../caffe/face-detection-retail-0004
+FACE_MODEL_ZOO_DIR = Retail/object_detection/face/sqnet1.0modif-ssd/0004/dldt/FP16
+FACE_FILE_NAME_BASE = face-detection-retail-0004
+
+AGE_GEN_RELATIVE_DIR = ../../caffe/AgeGenderNet
+AGE_GEN_MODEL_ZOO_DIR = Retail/object_attributes/age_gender/dldt/FP16
+AGE_GEN_FILE_NAME_BASE = age-gender-recognition-retail-0013
+
+
+.PHONY: data
+deps: 
```

Comment:
I think this should be `data: ` There was  a bug in another makefile that did this, you probably started with the buggy makefile?